src/main/java/com/lapask/FixedResizableHybridOverlay.java

```diff
@@ -10,58 +10,84 @@
 import net.runelite.client.ui.overlay.OverlayPosition;
 import net.runelite.client.util.ImageUtil;
 
-public class FixedResizableHybridOverlay extends Overlay
+
+public class FixedResizableHybridOverlay extends Overlay 
 {
-	private static final int OVERLAY_WIDTH = 249;
-
-	private final Client client;
-	//private final FixedResizableHybridPlugin plugin;
-	private final FixedResizableHybridConfig config;
-	private static final Image gapBorder = ImageUtil.loadImageResource(FixedResizableHybridPlugin.class, "/border15px.png");
-	@Inject
-	public FixedResizableHybridOverlay(Client client, FixedResizableHybridConfig config, FixedResizableHybridPlugin plugin)
-	{
-		this.client = client;
-		this.config = config;
-		//this.plugin = plugin;
-
-		// Set the overlay position and layer
-		setPosition(OverlayPosition.DYNAMIC);
-		// Render behind widgets
-		setLayer(OverlayLayer.UNDER_WIDGETS);
-	}
-
-	@Override
-	public Dimension render(Graphics2D graphics)
-	{
-		// Get the canvas height dynamically
-		Dimension clientDimensions = client.getRealDimensions();
-		int clientWidth = (int) clientDimensions.getWidth();
-		int clientHeight = (int) clientDimensions.getHeight();
-		Rectangle overlayBounds = new Rectangle(clientWidth - OVERLAY_WIDTH, 0, OVERLAY_WIDTH, clientHeight);
-
-		graphics.setColor(new Color(47, 42, 32));
-		graphics.fill(overlayBounds);
-		if (config.useGapBorders())
+
+    private static final int OVERLAY_WIDTH = 249;
+
+    private final Client client;
+		//private final FixedResizableHybridPlugin plugin;
```

Comment:
I would be careful with whitespace across the PR - both with line whitespaces and indents. Seems to be generating a lot more diff then necessary, and it's inconsistent with the original repo / sometimes unnecessary